### ISSUE TITLE
Základní login

### ISSUE THREAD

**Issue by White-Dolphin** [GeoMW/ClassicUO_Manawydan#17]
Poslední zmínka o verzi klasického klienta na MW je v dokumentaci "6.0.13".
Pokud se nastaví v launcheru tak pak vše funguje.
Po zprovoznění komunikace (pakety) bych zkusil jakou nejlepší verzi s našimi daty půjde použít a pro MW ji kódoval natvrdo (tj. úplně bych vypnul launcher).

**Comment by GeoMW** [comment]
Otázkou je, jestli verze klienta neovlivňuje dostupné featurz.

**Comment by White-Dolphin** [comment]
Ano, omezují. Bude potřeba vyzkoušet co náš server podporuje, a co už ne. Podle toho pak nastavit definitivní verzi, případně doplnit podporu na server.
